### ISSUE TITLE
Who owns the NPM package azure-streamanalytics-cicd?

### ISSUE THREAD

**Issue by oising** [Azure/azure-stream-analytics#122]
aka https://www.npmjs.com/package/azure-streamanalytics-cicd

We have an issue where the latest release 2.2.1 has broken our pipeline. Before I fill this out with details, can you direct me to the correct repo if this is not it. If it is the correct one, then reply back and I will continue here.

**Comment by sidramadoss** [comment]
Hi @oising , sorry about this. Our team is investigating and will fix it soon. A member of the team will respond to this thread once it is fixed.

**Comment by oising** [comment]
So is this the right repo for bugs about the tool? You could probably add this to the docs on npm and nuget ;)

**Comment by jfcroteau** [comment]
I think @oising meant version 2.2.6, the latest release.

**Comment by oising** [comment]
> I think @oising meant version 2.2.6, the latest release.

Oops, yes.

**Comment by pengyuli-ms** [comment]
Hi @oising, thanks for helping us improve the cicd tool!
The right repo for this bug is: https://github.com/microsoft/vscode-asa/issues

Would you mind move this bug to that repo? we could discuss further there.

**Comment by pengyuli-ms** [comment]
And BTW, could you share with me a sample ASA job you're using, so I could try to repro this issue on my side?

**Comment by oising** [comment]
@pengyuli-ms Sorry, we won't be able to share our job since it relies on data that is private. But it seems that the issue is more fundamental than data. The tool has a runtime error with some kind of assembly reference being missing (Microsoft.Build.dll)

**Comment by pengyuli-ms** [comment]
@oising Sure, and I noticed you're running the CICD command using a VS project, and actually the ASA VS extension is out of date, could you use [ASA VSCode extension](https://marketplace.visualstudio.com/items?itemName=ms-bigdatatools.vscode-asa) to create a same project? It supports more features and has less bugs.
(you could reference this issue discussion: https://github.com/microsoft/vscode-asa/issues/65#issuecomment-1105948735 for more detailed information.)

**Comment by kipusoep** [comment]
Same issue here. I see no issue has been created @ https://github.com/microsoft/vscode-asa/issues ?

**Comment by pengyuli-ms** [comment]
Yep @kipusoep, could you help create one in that repo with your issue?
